packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -73,45 +75,94 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
          * @returns {Promise<Invitation<OperatorKit>>}
          */
         makeOperatorInvitation(operatorId) {
-          const { admin } = this.facets;
+          const { creator } = this.facets;
           trace('makeOperatorInvitation', operatorId);
 
           return zcf.makeInvitation(
             /** @type {OfferHandler<OperatorKit>} */
             seat => {
               seat.exit();
-              return admin.initOperator(operatorId);
+              return creator.initOperator(operatorId);
             },
             INVITATION_MAKERS_DESC,
           );
         },
+        /** @param {string} operatorId */
+        async initOperator(operatorId) {
```

Comment:
I don't see an `await` in here. Why `async`? Do we expect it to grow in that direction?

Reply:
copypasta methinks. I don't see a reason it would ever need to be so I'll remove it.